src/App.js

```diff
@@ -61,7 +62,11 @@ const router = createBrowserRouter([
   },
   {
     path: "/bucket-list",
-    element: <BucketList />,
+    element: (
+      <BucketList>
+        <BucketForm />
```

Comment:
Are you planning to add bucket form as a nested directory? like /bucket-list/bucket-form. If so - might need to put as a child element, otherwise not needed to add this to App.js if a nested directory is optional!